boards/shelly_1

```diff
@@ -0,0 +1,32 @@
+{
+  "build": {
+    "arduino": {
+      "ldscript": "eagle.flash.1m.ld"
+    },
+    "core": "esp8266",
+    "extra_flags": "-DESP8266 -DARDUINO_ARCH_ESP8266 -DARDUINO_ESP8266_SONOFF_S20",
+    "f_cpu": "80000000L",
+    "f_flash": "40000000L",
+    "flash_mode": "dout",
+    "mcu": "esp8266",
+    "variant": "shelly"
```

Comment:
User won't be able to compile any project with this board since there is no such variant `shelly` in the Arduino core for ESP8266 https://github.com/esp8266/Arduino/tree/master/variants, but for some reason you specified `-DARDUINO_ESP8266_SONOFF_S20`. If the `Shelly` board is compatible with `Sonoff S20` then you can use `itead` variant, pinout might be different though.